electron/main.ts

```diff
@@ -262,20 +262,53 @@ ipcMain.on('quit', async () => handleExit())
 /* const storage: Storage = mainWindow.localStorage
 const lang = storage.getItem('language') */
 
-ipcMain.handle('getGameInfo', async (event, game) => {
+const fetchGraphQl = async (namespace: string, game:string) => {
```

Comment:
Function name is too Ambiguous. Having a more descriptive function name would make it easier to understand what the function does. Like getGameCatelogSlug or something like that.